raiden/tests/integration/transfer/test_directransfer.py

```diff
@@ -64,6 +65,7 @@ def test_direct_transfer_to_offline_node(raiden_network, token_addresses, deposi
         identifier=payment_identifier,
     )
 
+    gevent.sleep(2)
```

Comment:
Same as above. Arbitrary sleep values make tests unreliable.

Reply:
Again, everything here should be synchronous, this is just to be a little more sure about the state (e.g. here it should ensure the invite sent by the the online node was sent while the offline one was still offline, so it would need to pick this event only when it became online).